src/reminders/reminders.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable, Logger } from '@nestjs/common'
 import { Cron, CronExpression } from '@nestjs/schedule'
 
-import { Reminder, Prisma } from '@prisma/client'
+import { Reminder, Prisma, prisma } from '@prisma/client'
```

Comment:
Looks like this might've been imported by accident?

```suggestion
import { Reminder, Prisma } from '@prisma/client'
```